src/datasources/fault-ds/partials/config.html

```diff
@@ -1,2 +1,44 @@
 <datasource-http-settings current="ctrl.current">
 </datasource-http-settings>
+
+<h3 class="page-heading">OpenNMS FM Datasource Details</h3>
+<div class="gf-form-group">
+    <div class="gf-form-inline">
+        <gf-form-switch class="gf-form"
+                        label="Use grafana user"
+                        tooltip="Defines if on supported alarm operations the currently logged in
+                                 Grafana user or the datasource user is used to talk to the ReST endpoint.
+                                 Supported operations are escalating, clearing, un-/acknowleding alarms as
+                                 well as creating/updating a journal/memo.
+                                 NOTE: Only OpenNMS admin users are allowed to perform actions as another user."
+                        checked="ctrl.current.jsonData.useGrafanaUser"
+                        label-class="width-11"
+                        switch-class="max-width-6">
+        </gf-form-switch>
+
+        <div class="gf-form" ng-show="ctrl.current.jsonData.useGrafanaUser">
+            <label for="{{ctrl.id}}-grafana-user-field" class="gf-form-label pointer">
+                Field
+                <info-popover mode="right-normal" position="top center">
+                    Defines which field of the Grafana User should be used.
+                    This allows to fine grane what shows up as the "Action-Performing User" in OpenNMS. <br/>
```

Comment:
I would rephrase this as:
This allows for fine grained control over the value displayed as the "Action-Performing User" in OpenNMS.